demo.yml

```diff
@@ -50,36 +42,46 @@ x-services:
       - internal
     env_file:
       - ./demo-default.env
-      - ./demo-local.env
+      - ./demo-${COMPOSE_PROJECT_NAME}.env
+      - ./demo-${COMPOSE_PROJECT_NAME}-local.env
 
   base_api: &base_api_service
     <<: *base_service
     networks:
       - apis_external
       - internal
 
+
 services:
-  # service providers - minio, elasticMQ, postgres
+  ##
+  ## service providers - minio, elasticMQ, postgres
+  ##
+
+  # needed only for easy channels development - because on Linux you can't
+  # access the host container in any other reliable way
+  # And shared channels are usually started on the host for the local/demo
+  docker-host:
+    image: qoomon/docker-host
+    cap_add: [ 'NET_ADMIN', 'NET_RAW' ]
+    restart: on-failure
+    networks:
+      - internal
+
   postgresql:
     image: postgres:10.5
     environment:
       - POSTGRES_USER=intergovuser
       - POSTGRES_PASSWORD=intergovpassword
-    # we don't have to expose it directly to the host machine for our demo,
-    # but it's useful for starting service on the host
-    # ports: ['5432:5432']
     networks:
       - internal
+    volumes:
```

Comment:
I turned logging back on and, as mentioned in a comment above, local path volumes on Windows don't get proper permissions so named volumes is the only way to get postgres to run but not lose the contents of the volume if you stop and start the services.

Reply:
okay, sorry. I'm fine with the docker volumes as well - it was just useful for me to have it persistent even when all other volumes are deleted (on some stage)

Reply:
are you okay with the minio on windows? It's very comfortable for me to have that path shared with the host machine because I can empty and check the repost content easily using my  file browser, but if you hit the same issue then please do it the same - thus removing any need in `var*` directories.